packages/juce-blueprint/src/lib/BlueprintRenderer.ts

```diff
@@ -125,22 +110,20 @@ const HostConfig = {
   /** During a state change, this method will be called to identify the set of
    *  properties that need to be updated. This is more-or-less an opportunity
    *  for us to diff our props before propagating.
-   *
```

Comment:
I'm assuming removing the JSDoc param types makes sense here? They don't seem to add anything with the introduction of TS. Correct me if I'm wrong though @nick-thompson and I'll back this out.

Reply:
Yup, LGTM